content/blog/ferret-v0.11.0.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Ferret v0.11.0"
+subtitle: "More features"
+draft: false
+author: "Vladimir Fetisov"
+authorLink: "https://github.com/3timeslazy"
+date: "2020-05-19"
+---
+
+Hello fellow miners, **[Ferret v0.11.0](https://github.com/MontFerret/ferret/releases/tag/v0.11.0)** has been **finally** released!
+
+In this release, we mostly focused on our full-time jobs and families.
+
+But despite this, We have done a lot of cool features and projects.
+
+Let’s take a look!
+
+---
+
+# What's added
+## USE statement
+There is no need to write a full function path anymore! Just add ``USE PATH::TO::PACKAGE`` at the top of your query.
+ 
+{{< code fql >}}
+USE IO::FS
+
+LET favicon = DOWNLOAD('https://www.google.com/favicon.ico')
+
+// 'WRITE' the same as 'IO::FS::WRITE'.
+// RETURN IO::FS::WRITE('google.favicon.ico', favicon) also valid.
+RETURN WRITE('google.favicon.ico', favicon)
+{{</ code >}}
+
+## PATH functions
+A little bit functions to work with path strings.
```

Comment:
``New functions for working with file paths.``

Reply:
done